python/cuml/internals/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+
+
+set(cython_sources
+    internals.pyx
+)
+
+set(linked_libraries cuml::${CUML_CPP_TARGET})
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}"
+  MODULE_PREFIX internals_
+)
+
+foreach(cython_module IN LISTS RAPIDS_CYTHON_CREATED_TARGETS)
+  set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN;\$ORIGIN/cpp")
+endforeach()
+
+# We need to include for callbacks_implements.h in the internals folder
+target_include_directories(internals_internals PRIVATE "")
```

Comment:
I've never tried this before, is the current directory implicit in `target_include_directories`? Otherwise maybe this include isn't actually necessary?
```suggestion
target_include_directories(internals_internals PRIVATE "${CMAKE_CURRENT_LIST_DIR}")
```

Reply:
Current directory isn't includeded by default. I missed this patter but we should use `CMAKE_CURRENT_LIST_DIR` instead of ""